### PR TITLE
 Update JSX Elements and Styling to Match Zeplin's Design

### PR DESCRIPTION
**Pull request by gaks1**
# Description: 
This pull request updates the JSX elements and styling to match the design provided in Zeplin.

## Changes:

- [x]  Add missing JSX elements as per Zeplin's design
- [x]  Update existing JSX elements to match design
- [x]  Apply CSS rules from Zeplin to style the website
- [x]  Ensure final design matches Zeplin's design, including all elements and styling
- [x]  Verify the project requirements are met

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              name="author"
+              placeholder="Author"
+            />
+          </label>
+          <br />
+          <button type="submit" className="ADD-BOOK">
+            Add Book
+          </button>
+        </form>
+      </div>
```
Comment (danmainah) [inline src/components/Books.js:69]: - [ ] @gaks1 Great work on your design however the input section width is bigger than other elements, which makes it not match the zeplin design given kindly reduce the width to match the other elements' width.
image

Comment (danmainah) [reply src/components/Books.js:69]: - [ ] @gaks1 Great work on your design, however, your button width is smaller than the design given. also, the author's width is bigger than in the design, check the comparison below

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

File: src/components/Book.js [hunk 2]
```diff
@@ -1,61 +1,67 @@
...
+              aria-valuemax="100"
+            />
+            <div className="progress-text">
+              <p className="Percent">
+                57
+                <span>%</span>
+              </p>
+              <p className="Completed">Completed</p>
+            </div>
+          </div>
```
Comment (danmainah) [inline src/components/Book.js:43]: - [ ] @gaks1 Great work on your design, kindly ensure the progress bar circle color and thickness match the design given, below is a comparison

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @gaks1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @gaks1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**